trpc/queries/goals.ts

```diff
@@ -472,6 +481,11 @@ export const calcAchievedWeight = (
 ): number => {
     const { achivedWithWeight, comletedWithoutWeight, anyWithoutWeight, allWeight } = list.reduce(
         (acc, value) => {
+            // `where` filter by `deleted` field doesn't work in *Many queries
+            if (value.deleted) {
```

Comment:
shouldn't it be now `archived`?

Reply:
this field for criteria table, i wont rename this field in this pr :)